Docs: Explain how to hide X-rays and Our data sections (#12183)
metabase/metabase: docs/users-guide/01-what-is-metabase.md

Fresh out of the box, Metabase will show you a few things on the home page:

> - Some [automatic explorations](14-x-rays.md) of your tables that you can look at and save as a dashboard if you like any of them. It is possible to hide these suggestions for all users, while still keeping X-rays enabled, by hovering the section and clicking the `X` next to "Try these x-rays based on your data", when logged in as an administrator.

Reviewer: ```suggestion
- Some [automatic explorations](14-x-rays.md) of your tables that you can look at and save as a dashboard if you like any of them. (Administrators can permanently hide this section by clicking the `X` that appears on hover.)
```

- An area where things you or your teammates create will show up, along with a link to see all the dashboards, questions, and pulses you have.

> - A list of the databases you've connected to Metabase. It is possible to hide this section for all users by hovering the section and clicking the `X` next to "Our data", when logged in as an administrator.

Reviewer: ```suggestion
- A list of the databases you've connected to Metabase. (As with x-rays, administrators can permanently hide this section by clicking the `X` that appears on hover. You can always click on "Browse Data" from the main nav bar to see your databases and tables.)
```
